pr.codecept.js

```diff
@@ -39,13 +69,13 @@ exports.config = {
     },
     REST: {
       endpoint: process.env.PMM_UI_URL || 'http://127.0.0.1/',
-      timeout: 30000,
+      timeout: 3000,
```

Comment:
I don't think this would work, maybe UI timeouts might work but for API requests resp. might not be enough

Reply:
sure, will revert that